WcaOnRails/config/environments/development.rb

```diff
@@ -75,7 +75,7 @@
   config.active_storage.service = :local
 
   config.after_initialize do
-    Bullet.enable = !ENVied.DISABLE_BULLET
+    Bullet.enable = !EnvVars.DISABLE_BULLET?
```

Comment:
Why the question mark here but nowhere else?

Reply:
Oh yes there should be a question mark everywhere DISABLE_BULLET appears, I only added it here because this is the only place that stopped the server from booting

Reply:
just for clarification, without the question mark an exception would be thrown: 
`undefined method 'DISABLE_BULLET', did you mean DISABLE_BULLET?`

Reply:
I'm only speculating here because it's not explicitly written in the readme, but I think boolean values are automatically appended with a `?` to match ruby's convention that methods returning bool end with a `?`.
Actually I looked it up during my review, this is indeed what is being done [here](https://github.com/fnando/env_vars/blob/main/lib/env_vars.rb#L35).

Reply:
Thanks, that totally makes sense!